aws-auth-cognito/src/main/java/com/amplifyframework/auth/cognito/actions/SignInCustomActions.kt

```diff
@@ -74,7 +74,7 @@ object SignInCustomActions : CustomSignInActions {
                     )
                 }
             } catch (e: Exception) {
-                AuthenticationEvent(AuthenticationEvent.EventType.CancelSignIn(error = e))
+                SignInEvent(SignInEvent.EventType.ThrowError(e))
```

Comment:
Throw error to return error from API and cancel sign in to rest AuthN state.

Reply:
done.